### DISCUSSION TITLE
Higher UI scales are... _something_

### DISCUSSION THREAD

**Discussion by Tenexxt** [ppy/osu#29956] [Q&A]
Basically, i _highly dislike_ how they work in certain situations.

My personal favorite example of why do i think this needs to have some sort of adjustments is the daily challenge menu:
![{B3983A26-F242-4BCC-88CD-48E63755AFD2}](https://github.com/user-attachments/assets/6d56b658-bfd3-4259-9094-ce1e759d6d31)

This is from the highest UI scale possible in the game btw.
The score counts aren't even visible at all (and are very unreadable in some smaller scales). I could pass most elements here, since UI scailing exists to make things literally bigger, so it makes sense for most UI to scale the same, but I feel like the Header text and Beatmap display (mostly the header, but i feel like the Beatmap display is too big too a bit) are taking to much space, with not even their design but spacing

Profile page is breaking:
![{487EE666-1C2B-40F5-9616-AB0AD0ACE6BE}](https://github.com/user-attachments/assets/4ef1d299-c9db-462c-adbe-0814091a4c88)
![{11416F8B-C296-410E-9C54-26ACE850DE6D}](https://github.com/user-attachments/assets/14ccd281-60d6-48c4-8f7b-40dde73c37ff)
![{EDF348F1-AD6A-49D2-B64E-699E4BEC6D2C}](https://github.com/user-attachments/assets/48caa61b-ef98-4a34-a458-b93255a7caa9)

Song Select:
![{4950CEA5-557F-44A1-94DD-821FB96BF757}](https://github.com/user-attachments/assets/a710b08f-141e-4da1-8770-a576b7231f19)

Since song select is getting reworked, this hopefully won't be an issue in the future, but why is the header/background? taking up so much space? Takes up a quarter of the screen for no reason, just because it looks good on smaller scales.
Im not sure how this will be handled IF it will be handled in the future song select, as ill admit i didn't look at it, but if this object stays, then it should scale differently.

I feel like some other things also could be changed, but unlike anything i listed they aren't really that visually breaking and need changing that much.

**Comment by peppy** [comment]
It's best-effort at the moment. There's a lot of places it will break, but the overall goal is to make the game usable in most places at higher scales to account for smaller screens.

For now you can try choosing a lower scale where it doesn't break as much.
